apps/dashboard/src/app/(app)/team/[team_slug]/[project_slug]/components/ProjectFTUX/ProjectFTUX.tsx

```diff
@@ -63,7 +63,7 @@ function IntegrateAPIKeySection({
           {secretKeyMasked && (
             <SecretKeySection
               secretKeyMasked={secretKeyMasked}
-              projectId={project.id}
+              project={project}
```

Comment:
prefer passing `projectId` and `teamId` separately instead since later when we pass from server component to client that is a lot less data to serialize across server -> client